joomla/joomla-cms
# Adapt text and bootstrap size in blog sample data

Pull Request for Issue #33832 .

### Summary of Changes
Change text to "To totally change the look of your site you will probably want to install a new template. **Go to System, select Install - Extensions from the list and the extension installer will open**. There are many free" 

Bootstrap size of module site information is set to 0, to fit recently made changes in dashboard modules. #33045 et alii

### Testing Instructions
Code review or install blog sample data, 
Front end : Log in and and check the text "Working on your Site"
Backend:  Check the with of the module site information


### Actual result BEFORE applying this Pull Request
Module site information has boostrap size 6 and has only half the width of the column
Text see  #33832 .

### Expected result AFTER applying this Pull Request
Module site information has same size as other modules
Text is correct for J4

### Documentation Changes Required
